### ISSUE TITLE
引っ越し

### ISSUE THREAD

**Issue by supermomonga** [supermomonga/yatteiku#1]
:fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax:
## 解約手続きとか
- [ ] マンション管理会社に電話
  - [X] 電話
  - [ ] 書類返送
- [ ] フレッツ光の引っ越し手続き
  - [X] 申し込み
  - [x] 開通工事日決定
- [ ] 水道解約
- [ ] 電気解約
- [ ] ガス解約
## 引っ越し
- [ ] 要らないもの捨てる
- [ ] 引越し業者見積もり依頼 http://www.moving-s.com/
  - 2tショートトラックでいける？ http://www.moving-s.com/plans/tanshin/tanshin.php
  - 有料オプションどれ使うか http://www.moving-s.com/options/
    - ダンボール箱詰め http://www.moving-s.com/options/extra-services/packing-service.php
    - ダンボール荷解き（要る？） http://www.moving-s.com/options/extra-services/unpacking-service.php
## 新居各種手続き
- [X] 8日 鍵受け取り 新宿
- [X] 水道契約
- [X] 電気契約
## その他
- [ ] 郵便物転送
  - 手続きは転送先の郵便局？転送元の郵便局？
## 　完了後
- [ ] 引っ越し祝い.vim

:fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax:

**Comment by aycabta** [comment]
:heart_eyes::older_man:

**Comment by aycabta** [comment]
:fax:

:sound:→:mute:

**Comment by aycabta** [comment]
Let's :phone:!

**Comment by aycabta** [comment]
:house: + :fax: = :articulated_lorry:

**Comment by aycabta** [comment]
:cool:

**Comment by supermomonga** [comment]
:fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax:
:fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax:
:fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax:
:fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax:
:fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax: :fax:

**Comment by aycabta** [comment]
:u7121::u5272:

:vs:

:b:

**Comment by aycabta** [comment]
:curly_loop::fax:

**Comment by supermomonga** [comment]
1. :hamster: :package: :point_right: :articulated_lorry: 
2. :house:  :articulated_lorry: :dash:  
3. :hamster: :house: :clap:

**Comment by aycabta** [comment]
I don't think that "荷解き" for all "荷" is necessary, because you can determine un"解き"ed "荷" are dusts.

**Comment by supermomonga** [comment]
what dusts means :question:

**Comment by aycabta** [comment]
どうせ要らねえもん溜めてんだろってことだよ、ダンボールに詰めたままになるようなのは捨てろ。

**Comment by supermomonga** [comment]
荷造りするまでにほぼ捨てる予定

**Comment by aycabta** [comment]
But you must request "荷解き" for heavy "荷"s that make you "D".

Ex.) Moniter arm.

**Comment by supermomonga** [comment]
got it.

**Comment by supermomonga** [comment]
とりあえず必須のもの以外すべて捨てようかな、食器とか全然着てない服とか

**Comment by aycabta** [comment]
![OH, YOU!](http://i.imgur.com/0rIpk.png)

**Comment by supermomonga** [comment]
:dog: :smirk:

**Comment by aycabta** [comment]
![GOD FOUND WALLY](http://cdn.twentytwowords.com/wp-content/uploads/God-knows-where-Waldo-is.jpg?43c206)

**Comment by aycabta** [comment]
![THE ｸｩ〜ﾝ DOG](http://justsomething.co/wp-content/uploads/2014/05/dogs-caught-in-the-act-2.jpg)

THE ｸｩ〜ﾝ DOG

**Comment by aycabta** [comment]
![DOGE](http://c0.thejournal.ie/media/2013/12/doge-sun-meme.jpg)

**Comment by supermomonga** [comment]
@rhysd will post nice photos.

**Comment by aycabta** [comment]
![THE GIF](http://fc02.deviantart.net/fs70/f/2013/342/0/a/have_a_doge_day__cry_by_hinechan-d6x62v3.png)

This is PNG file.

**Comment by supermomonga** [comment]
I see.

**Comment by aycabta** [comment]
![THE GOD](https://fc07.deviantart.net/fs71/f/2013/247/e/0/doge_is_life_by_wowstahpplz-d6l2qfo.jpg)

ｸｩ〜ﾝ GOD, DOGE.

**Comment by aycabta** [comment]
![much infinity](http://i.imgur.com/q7VKiiK.gif)

![aieru-tan](https://pbs.twimg.com/profile_images/2622670832/24m43mi6jld39507h1it_400x400.png)

**Comment by aycabta** [comment]
![0](http://i.imgur.com/76yFaur.jpg)

![1](http://i.imgur.com/V9mVuO7.jpg)

![2](http://41.media.tumblr.com/9ef3d8bb5c0b501b4cbf282c2e3710bd/tumblr_mzv52aYOm41qeablwo1_500.jpg)

**Comment by rhysd** [comment]
:dog:

**Comment by aycabta** [comment]
WOW

**Comment by meta-closure** [comment]
はじめまして！！こんにちは！！！！
引っ越しですか？いいですね！プルリクエストを送りました！！

**Comment by supermomonga** [comment]
きてない（笑）

**Comment by rhysd** [comment]
- [ ] 引っ越し祝い momonga.vim

must be listed.

**Comment by supermomonga** [comment]
なるほど

**Comment by rhysd** [comment]
各タスクのデッドラインが無いですが，進捗どうですか？
